text/0000-buildpack-config-for-dist.md

```diff
@@ -0,0 +1,298 @@
+# Meta
+[meta]: #meta
+- Name: Buildpack Configuration for Distribution
+- Start Date: July 22, 2019
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal to change the schema of `buildpack.toml` to separate the
+concerns of defining a buildpack and managing configuration of buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+In the approval of the [Distribution Spec](https://github.com/buildpack/rfcs/blob/master/text/0007-spec-distribution.md),
+we landed on a [compromise that no one was really happy with](https://github.com/buildpack/rfcs/pull/11#issuecomment-510483638).
```

Comment:
I disagree with this characterization. I liked the flexibility provided by the original proposal, and the drawbacks mentioned below are not significant for my personal use cases / preferences.